src/axelrod_dojo/utils.py

```diff
@@ -138,16 +134,27 @@ def params(self):
     def crossover(self, other):
         pass
 
+    def receive_vector(self, vector):
```

Comment:
Can we have docstrings for the new functions? (I know all the old ones don't have them...)